### PR TITLE
IGVF-1641-audit-page-sorting

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: pages/audits.js [hunk 1]
```diff
@@ -78,20 +82,62 @@ export default function AuditDoc({ auditDoc }) {
...
     return {
       props: {
         auditDoc,
+        schemas,
         pageContext: { title: "Audit Documentation" },
       },
     };
   }
-  return errorObjectToProps(auditDoc);
+  return errorObjectToProps(auditDoc, schemas);
```
Comment (forresttanaka) [inline pages/audits.js:142]: `errorObjectToProps` only takes one parameter.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
Just a couple small changes.

File: pages/audits.js (hunk 1 above)
Comment (khzlin) [reply pages/audits.js:142]: done

File: pages/audits.js [hunk 2]
```diff
@@ -78,20 +82,62 @@ export default function AuditDoc({ auditDoc }) {
...
   auditDoc: PropTypes.object.isRequired,
+  // List of schemas to display in the list; directly from /profiles endpoint
+  schemas: PropTypes.shape({
+    _hierarchy: PropTypes.shape({
+      Item: PropTypes.object.isRequired,
+    }).isRequired,
+  }),
 };
 
+// Function that maps 1 type name to 1 collection Name
```
[the remark this answers is not in the document]
Comment (khzlin) [reply pages/audits.js]: done

**Review by forresttanaka — APPROVED** [review]

**Pull request merged**